minetest/minetest
# [no squash] Irrlicht import

First commit just copies irrlichtmt (as of https://github.com/minetest/irrlicht/commit/124708784fdc4321d5983cdbfb889e62fbd25f40) to `<root>/irr/`.

Second commit makes use of this irrlichtmt version, and removes a bunch of now useless stuff.

Related: #13642

Note: `lib/irrlichtmt` is still gitignored. This is good because you can still bisect this way, and also we don't mess with repos lying around there.
Users can decide themselves when they want to delete it, but they have to do it manually.

Co-authored by sfan5.

## To do

This PR is Ready for Review.

## How to test

* Build.
* Watch the CI.
* Check and test build artifacts.
* Read through other commits. Reproduce 1st commit if you want.
